allen-cell-animated/cell-feature-data
# Refactor/rename and gitignore

Problem
=======
What is the problem this work solves, including

- partially closes #105 
- pr size: small
- estimate review time: <10 mins

Solution
========
What I/we did to solve this problem
- replaced `python_dataset_creator` with the new name `cell_feature_data`
- reorganized `.gitignore` to ignore all virtual environments and egg-infos
- updated README accordingly    

## Type of change
Please delete options that are not relevant.

* refactoring and maintenance 


Steps to Verify:
----------------
1. The changes of renaming and editing `.gitignore` shouldn't affect any existing functionalities 
2. (optional)Follow the [instructions](https://github.com/allen-cell-animated/cell-feature-data/tree/refactor/rename-and-gitignore#option-1-use-the-cell_feature_data-package) in Readme to create a single dataset to check if the changes break anything or if you still see ~4k changes from the dependencies
